packages/core/theme/src/components/alert.ts

```diff
@@ -0,0 +1,94 @@
+import type {VariantProps} from "tailwind-variants";
+
+import {tv} from "../utils/tv";
+/**
+ * Alert wrapper **Tailwind Variants** component
+ *
+ * @example
+ * ```js
+ * const {base, mainWrapper, title, description} = input({...})
+ *
+ * <div className={base())}>
+ *    // icon
+ *
+ *    <div classsName ={mainWrapper()}>
+ *       <div className={title()}>Title</div>
+ *       <div className={description()}>Description</div>
+ *   </div>
+ *
+ *    <button>
+ *      // close button
+ *    </button>
+ * </div>
+ * ```
+ */
+
+const alert = tv({
+  slots: {
+    base: ["group flex flex-row w-[342px] h-[68px]  p-[12px]"],
```

Comment:
1. `group` -> `group/alert` so that it won't conflict with other components.
2. may double check if you really need group here or not

Reply:
Yeah we might not need it here. Will check it.